components/raftstore/src/store/fsm/peer.rs

```diff
@@ -106,8 +106,10 @@ pub struct PeerFsm<E: KvEngine> {
 
 impl<E: KvEngine> Drop for PeerFsm<E> {
     fn drop(&mut self) {
+        info!("peer fsm destroyed"; "region_id" => self.region_id());
         self.peer.stop();
         while let Ok(msg) = self.receiver.try_recv() {
+            info!("drop message after peer was destroyed"; "region_id" => self.region_id(), "msg" => ?msg);
```

Comment:
Please do not exceed 100 chars.

Reply:
I don't think the message is necessary. And `msg` may contains sensitive data.

Reply:
😅I will revert them.